index.html

```diff
@@ -4286,10 +4286,14 @@ <h2 id="AudioScheduledSourceNode">
                 time coordinate system as the
                 <a><code>AudioContext</code></a>'s <a href=
                 "#widl-BaseAudioContext-currentTime">currentTime</a> attribute.
-                If 0 is passed in for this value or if the value is less than
-                <b>currentTime</b>, then the sound will start playing
-                immediately. <span class="synchronous">A TypeError exception
-                MUST be thrown if <code>when</code> is negative.</span>
+                When the signal emitted by the <a>AudioScheduledSourceNode</a>
+                depends on the sound's start time, the exact value of
+                <code>when</code> is always used without rounding to the
+                nearest sample frame. If 0 is passed in for this value or if
+                the value is less than <b>currentTime</b>, then the sound will
+                start playing immediately. <span class="synchronous">A
+                TypeError exception MUST be thrown if <code>when</code> is
```

Comment:
I think we're going to use RangeError for this kind of stuff. However, we haven't really decided yet.